nextjs/src/modules/episodes/IndividualEpisodePage.js

```diff
@@ -35,6 +36,12 @@ const IndividualEpisodePage = ({
 }) => {
   // state
   const [skipTo, setSkipTo] = useState(null);
+  const router = useRouter();
+
+  useEffect(() => {
+    const UrlParams = router.query;
+    skipToTimestamp(UrlParams.time);
```

Comment:
Think I figured out at-least the base issue. This is a string. How it is here and how it was before. Need to be a number. `Number(UrlParams.time)` or how ever ya wanna do it. 

All I changed after pulling was:
```javascript
 const skipToTimestamp = (time) => {
    setSkipTo(Number(time));
  };
```

and it is now jumping. There are some other issues with with it now auto playing and not switching the play/pause icon

TS woulda helped here ;)

Reply:
This was the PropTypes for `WaveFormPlayer`

```javascript
WaveformPlayer.propTypes = {
  audioPath: PropTypes.string,
  episodeNumber: PropTypes.number,
  episodeTitle: PropTypes.string,
  skipTo: PropTypes.number,
};
```

Reply:
Math.floor() wants a number.

```typescript
floor(x: number): number;
    /**
     * Returns the natural logarithm (base e) of a number.
     * @param x A numeric expression.
     */
```